test/unittest/EntityMetatraffic/EntityMetatrafficTests.cpp

```diff
@@ -0,0 +1,120 @@
+// Copyright 2021 Proyectos y Sistemas de Mantenimiento SL (eProsima).
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <gtest_aux.hpp>
+#include <gtest/gtest.h>
+
+#include <database/entities.hpp>
+
+using namespace eprosima::statistics_backend::database;
+
+constexpr const char* PID_DEFAULT = "36000";
+constexpr const char* DATA_TYPE_DEFAULT = "data_type";
+#define QOS_DEFAULT Qos(R"({"qos":"empty"})"_json)
+
+
+#define LOCATOR_DEFAULT_NAME(x) "locator_" + std::to_string(x)
+#define HOST_DEFAULT_NAME(x) "host_" + std::to_string(x)
+#define USER_DEFAULT_NAME(x) "user_" + std::to_string(x)
+#define PROCESS_DEFAULT_NAME(x) "process_" + std::to_string(x)
+#define DOMAIN_DEFAULT_NAME(x) "12" + std::to_string(x)
+#define ALIAS_DEFAULT_NAME(x) "domain_" + std::to_string(x)
+#define TOPIC_DEFAULT_NAME(x) "topic_" + std::to_string(x)
+#define PARTICIPANT_DEFAULT_NAME(x) "participant_" + std::to_string(x)
+#define DATAWRITER_DEFAULT_NAME(x) "datawriter_" + std::to_string(x)
+#define DATAREADER_DEFAULT_NAME(x) "datareader_" + std::to_string(x)
+
+#define GUID_DEFAULT(x) "01.0f.00.00.00.00.00.00.00.00.00.0" + std::to_string(x) + "|0.0.0.0"
+
```

Comment:
Add a tests that checks only the `is_metatraffic_topic` (unittest). It is not needed to have a database or anything, just to create a bunch of strings and check that everyone of them is classified correctly